src/main/webapp/app/modeling-assessment-editor/modeling-assessment-conflict/modeling-assessment-conflict.component.ts

```diff
@@ -1,45 +1,76 @@
-import { AfterViewInit, Component, Input, OnInit } from '@angular/core';
+import { AfterViewInit, Component, OnInit } from '@angular/core';
 import { ModelingSubmission, ModelingSubmissionService } from 'app/entities/modeling-submission';
-import { ModelingExercise } from 'app/entities/modeling-exercise';
-import { ActivatedRoute } from '@angular/router';
+import { ActivatedRoute, Router } from '@angular/router';
 import { UMLModel } from '@ls1intum/apollon';
 import { Conflict, ConflictingResult } from 'app/modeling-assessment-editor/conflict.model';
 import { AccountService, User } from 'app/core';
 import * as $ from 'jquery';
+import { JhiAlertService } from 'ng-jhipster';
+import { ModelingExercise } from 'app/entities/modeling-exercise';
+import { Feedback } from 'app/entities/feedback';
+import { ConflictResolutionState } from 'app/modeling-assessment-editor/conflict-resolution-state.enum';
+import { ModelingAssessmentService } from 'app/entities/modeling-assessment';
 
 @Component({
     selector: 'jhi-modeling-assessment-conflict',
     templateUrl: './modeling-assessment-conflict.component.html',
-    styleUrls: ['./modeling-assessment-conflict.component.scss'],
+    styleUrls: ['./modeling-assessment-conflict.component.scss', '../modeling-assessment-editor.component.scss'],
 })
 export class ModelingAssessmentConflictComponent implements OnInit, AfterViewInit {
```

Comment:
As somebody who doesn't know so much about the conflict assessment, I would really like more JS Doc :)
Some methods are self explanatory, but e.g. I wonder what happens onFeedbackChanged without reading the code.

Reply:
I agree, please add some documentation as this conflict stuff is rather complicated.

Reply:
Since the complete component gets splitted and refactored in the conflict notification branch, i will add some documentation to the ModellingAssessmentComponent and the components that derive from it there.